PredicateKitTests/OperatorTests.swift

```diff
@@ -1335,6 +1335,49 @@ final class OperatorTests: XCTestCase {
     XCTAssertEqual(comparison.operator, .in)
     XCTAssertEqual(value, ["hello", "world", "welcome"])
   }
+    
+  func testKeyPathInArray() throws {
+    let predicate: Predicate<Data> = (\Data.text).in(["hello", "world", "welcome"])
+
+    guard case let .comparison(comparison) = predicate else {
+      XCTFail("text.in(['hello', 'world', 'welcome']) should result in a comparison")
+      return
+    }
+
+    guard let keyPath = comparison.expression.as(KeyPath<Data, String>.self) else {
+      XCTFail("the left side of the comparison should be a key path expression")
+      return
+    }
+
+    let value = try XCTUnwrap(comparison.value as? [String])
+
+    XCTAssertEqual(keyPath, \Data.text)
+    XCTAssertEqual(comparison.operator, .in)
+    XCTAssertEqual(value, ["hello", "world", "welcome"])
+  }
```

Comment:
I think this is testing [this pre-existing function](https://github.com/ftchirou/PredicateKit/blob/main/PredicateKit/Predicate.swift#L483). That would explain why the [code coverage](https://github.com/ftchirou/PredicateKit/suites/5737924361/artifacts/190255986) went down. I would suggest to test on a different property. Something like:

```swift
let predicate: Predicate<Data> = (\Data.count).in([21, 42, 63])
/// ...
```